dev/com.ibm.ws.concurrent.persistent_fat_multiple/fat/src/com/ibm/ws/concurrent/persistent/fat/multiple/DatabaseContainerUtil.java

```diff
@@ -0,0 +1,149 @@
+package com.ibm.ws.concurrent.persistent.fat.multiple;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.Statement;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+import com.ibm.websphere.simplicity.config.DataSource;
+import com.ibm.websphere.simplicity.config.DataSourceProperties;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.websphere.simplicity.config.dsprops.Properties;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_db2_jcc;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_microsoft_sqlserver;
+import com.ibm.websphere.simplicity.config.dsprops.Properties_oracle;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.topology.impl.LibertyServer;
+
+public class DatabaseContainerUtil {
+	//Logging Constants
+	private static final Class<DatabaseContainerUtil> c = DatabaseContainerUtil.class;
+	private static final String nl = System.getProperty("line.separator");
+	
+	/**
+	 * Initializes SQL Server database.  Creates a database with a given name, and
+	 * runs the sp_sqljdbc_xa_install stored procedure to enable XA transactions.
+	 * 
+	 * @param cont - SQLServer Test Container
+	 * @param databaseName - Name to be given to database
+	 */
+	public static void initSQLServer(JdbcDatabaseContainer<?> cont, String databaseName) {
```

Comment:
the `databaseName` param can likely be removed. I believe we can get the same info inside the method by calling `cont.getDatabaseName()`

Reply:
also, lets just rename this method to `initDatabase` and have it no-op if the DB type is _not_ SQLServer. This way the FAT suite doesn't need to check what the DB type is and conditionally call this method